apcd-cms/src/apps/view_users/templates/view_users.html

```diff
@@ -53,15 +53,19 @@ <h1>View Users</h1>
 <script>
 /*filters by organization with dropdown*/
 function filterTableByOrganization() {
-  var dropdown, filter, table, tr, td, i;
+  var dropdown, filter, newinput, bestinput, table, tr, td, i;
   input = document.getElementById("organizationFilter");
-  filter = input.value.toUpperCase();
+  newinput = input.value.replace("&","");
```

Comment:
I wonder if here, we can preprocess the org_names for the filter in the view for this page (to eliminate duplicates, extra characters, etc) so we can just leave str comparison for this function. I only suggest because I see on the staging page that there are some orgs in the dropdown that are the same, just different cases in their names (Ie the org name is both camel-cased and in all lowercase, but both are in the dropdown). What do you think?

Reply:
Will have this as a follow on task as per 2.10 scrum